pytensor/link/jax/dispatch/random.py

```diff
@@ -251,6 +251,28 @@ def sample_fn(rng, size, dtype, *parameters):
     return sample_fn
 
 
+@jax_sample_fn.register(aer.HalfNormalRV)
+def jax_sample_fn_halfnormal(op):
+    """JAX implementation of `HalfNormalRV`."""
+
+    def sample_fn(rng, size, dtype, *parameters):
+        rng_key = rng["jax_state"]
+        rng_key, sampling_key = jax.random.split(rng_key, 2)
+        (
+            loc,
+            scale,
+        ) = parameters
+        sample = (
+            loc
+            + jax.random.truncated_normal(sampling_key, 0.0, jax.numpy.inf, size, dtype)
```

Comment:
This seems inefficient, we can just take the absolute of the normal centered at 0
```suggestion
            + jax.numpy.abs(jax.random.normal(sampling_key, size, dtype))
```